product_brand/reports/account_invoice_report.py

```diff
@@ -11,14 +12,14 @@ class AccountInvoiceReport(models.Model):
 
     @api.model
     def _select(self):
-        select_str = super()._select()
+        select_str = super()._select().code
```

Comment:
Isn't there a way of not having to unwrap/wrap this? I suppose there should be a method to add new content to the existing SQL object.